extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/config/GrpcServerConfiguration.java

```diff
@@ -65,4 +65,10 @@
      */
     @ConfigItem(defaultValue = "false")
     public boolean enableReflectionService;
+
+    /**
+     * Number of gRPC server verticle instances.
```

Comment:
I would add a sentence about the scaling. It's not obvious.

Reply:
Updated